env-bootstrap/src/lib.rs

```diff
@@ -73,9 +73,11 @@ pub fn fixup_appimage() {
             clean_wezterm_config_env();
         }
 
-        if config::CONFIG_DIR.starts_with(append_extra_file_name_suffix(&appimage, ".config")) {
-            std::env::remove_var("XDG_CONFIG_HOME");
-            clean_wezterm_config_env();
+        for dir in config::CONFIG_DIRS.iter() {
+            if dir.starts_with(append_extra_file_name_suffix(&appimage, ".config")) {
+                std::env::remove_var("XDG_CONFIG_HOME");
```

Comment:
The purpose of this block was to check to see if XDG_CONFIG_HOME is controlled/manipulated by appimage. I don't think it makes sense to unset XDG_CONFIG_HOME here if a secondary directory under XDG_CONFIG_DIRS happens to match this criteria.

Perhaps the `for dir in CONFIG__DIRS` would be better off actually checking `std::env::var('XDG_CONFIG_HOME`)` directly?

Reply:
That makes sense. My new changes should reflect this!